packages/gateway/src/features/registration/schema.graphql

```diff
@@ -356,6 +356,7 @@ type DeathRegistration implements EventRegistration { # -> Composition
   medicalPractitioner: MedicalPractitioner # -> Encounter.participant
   createdAt: Date # -> .date($first)
   updatedAt: Date # -> .date
+  history: [History]
```

Comment:
Maybe we can put this in `EventRegistration` as this common for both events

Reply:
absolutely